src/api/app/models/staging/workflow.rb

```diff
@@ -22,7 +22,8 @@ def stageable
     end
 
     def ready_to_stage
```

Comment:
this now duplicates 90% of the `stagable` code - and taking that both scopes are called in one function each, which duplicates even more code... Perhaps this can be better expressed with a parameter to the state review?